sstables/sstables.cc

```diff
@@ -1183,6 +1183,78 @@ void sstable::set_position_range() {
     _position_range = position_range(pip(min_elements, bound_kind::incl_start), pip(max_elements, bound_kind::incl_end));
```

Comment:
Commit log might be out-of-date, it talks about storing first/last row in the sstable. This method doesn't store anythiing, it just looks up the first/last positions respectively.

Reply:
Will change. thanks